tests/api/test_projects.py

```diff
@@ -505,9 +506,10 @@ def test_invalid_id(self, session, flask_api_url):
     def test_success(self, session, flask_api_url):
         assert True
 
-    def test_unauthorized_update(self, flask_api_url):
+    def test_unauthorized_update(self, flask_api_url, enter_default_user_users):
         """ Tests for 403 when attempting to update a different users project
         """
+        response = enter_default_user_users
```

Comment:
we could check for the response to be correct here but i just assume it is working correctly